.github/workflows/build.yml

```diff
@@ -0,0 +1,61 @@
+name: build
+
+on:
+  push:
```

Comment:
should we leave that if it's empty?

Reply:
I think it is to have have it enable for any push